Sources/AppleAPI/Client.swift

```diff
@@ -56,6 +61,90 @@ public class Client {
             }
     }
     
+    public func srpLogin(accountName: String, password: String) -> Promise<Void> {
```

Comment:
If this references an implementation or specific other specification, it would be nice to link it as part of a comment for what/how you followed to implement this